androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/Settings.kt

```diff
@@ -42,9 +44,13 @@ fun Settings(
     onDismissRequest: () -> Unit,
     onSpeedSelected: (Float) -> Unit,
     onIsLoopEnabledSelected: (Boolean) -> Unit,
+    selectedQualityTrack: TrackQualityItem,
```

Comment:
Shouldn't we wrap this into a lambda to avoid recomposing?

Reply:
DOne